app/controllers/api/account.php

```diff
@@ -185,9 +186,11 @@
             throw new Exception(Exception::USER_BLOCKED); // User is in status blocked
         }
 
+        $sessionDuration = ($project->getAttribute('sessionDuration', 0) * 60) ?? Auth::TOKEN_EXPIRATION_LOGIN_LONG;
```

Comment:
```suggestion
        $duration = ($project->getAttribute('sessionDuration', 0) * 60) ?? Auth::TOKEN_EXPIRATION_LOGIN_LONG;
```

Reply:
same for others. when we can and when not used as a namespaces in other areas, let's prioritize a single word as the name.